pkg/workflow_template.go

```diff
@@ -346,6 +369,28 @@ func (c *Client) GetWorkflowTemplate(namespace, uid string, version int32) (work
 	return
 }
 
+func (c *Client) GetWorkflowTemplateByName(namespace, name string, version int32) (workflowTemplate *WorkflowTemplate, err error) {
```

Comment:
Why do you need a version if you're getting a template by name @aleksandrmelnikov ?